js/modules/k6/ws/ws.go

```diff
@@ -108,18 +108,17 @@ func (*WS) Connect(ctx context.Context, url string, args ...goja.Value) (*WSHTTP
 		return nil, errors.New("last argument to ws.connect must be a function")
 	}
 
-	// Leave header to nil by default so we can pass it directly to the Dialer
-	var header http.Header
-
+	header := make(http.Header)
 	tags := state.CloneTags()
 
+	header.Set("User-Agent", state.Options.UserAgent.String)
```

Comment:
Can you move this before `CloneTags`, please? I would prefer to not have it in the middle between initialization and assignment.

Reply:
Sure.! Working on that.